packages/styleguides/docs/src/05.flavors.md

```diff
@@ -0,0 +1,101 @@
+
+In the end, users and stakeholders will benefit from this. By using the same set of components (tested for accessibility and usability) we will improve efficiency, support DGs in their online communication and increase the quality of the whole «Europa Web Family».
+
+### High-level changes in ECL
+
+Here's a brief list of changes done in the ECL's project repository in order to accommodate the new EU-flavor components:
+
+* The single [fractal configuration](https://fractal.build/guide/core-concepts/configuration-files) rendering a style guide for ECL compnents is now split into 3 different configuration files. All 3 sub-projects are managed under `./packages/styleguides` folder.
+* Existing ECL components are moved as-is from `./framework` folder in a sub-folder `./src/flavors/ec`.
+* EU-flavor components are placed into `./src/flavors/eu`.
+* Visual regression testing configuration accept an optional `FLAVOR` environment variable. For further details, please check visual regression testing documentation pages.
+
+### Development notes
+
+The fact that ECL is to accommodate 2 separate sets of style guides, for each flavor, means that each flavor-specific style guide needs to be a separate fractal instance. Being a separate fractal instance bring these benefits:
+
+* Concrete modifications in one flavor-specific style guide visualization will not affect the other style guide.
+* Build process is straight-forward, an [default build capabilities of fractal](https://fractal.build/guide/project-settings) are used
+* Work on components from a given flavor is independent from the work on components of the other flavor
+
+At the same time, splitting work in per-flavor focus also means there are constraints:
+
+* Templates of components in one flavor cannot be used by templates of the other flavor, i.e. `fractal.components.set` configuration option.
+
+In short, this means that each flavor will have an independent set of:
+
+* project configurations for title, statuses, etc. - anything defined in `fractal.js`
+* components' templates
+* `ecl-builder.config.js` configurations about how CSS, JS and other assets are managed
+* each project lives and progresses independently from the other
+
+### Maintaining flavors (Important notes on APIs)
+
+The split of the project introduces several important rules which should be followed in order to keep ECL project repository maintainable for contributors, and also easy to integrate by consumers of ECL builds and resources.
+
+#### Naming
+
+Each flavor should be named in a distinguishable manner:
+
+* In `package.json` one would have `@ec-europa/eu-logos`. `@ec-europa` namespace should be preserved for correct organization on [npmjs](https://www.npmjs.com/). The `eu-logos` is for the EU flavor, whereas `ec-logos` should be the one for the EC flavor.
```

Comment:
I'm not sure that it's the best way to name packages. `@ec-europa/eu-logos` doesn't tell you the package is built within the ECL. I suggest `@ec-europa/ecl-logos-eu`, `@ec-europa/ecl-logos--eu` or, shorter,  `@ecl/eu-logos` or `@ecl/logos-eu`. Feel free to add more suggestions and we, as a team, will decide what naming convention to follow.

ping @degliwe @raduchiriac @emeryro